README.md

```diff
@@ -118,14 +118,20 @@ add the `ServiceAccount`, `ImageStream`, and `DeploymentConfig` from
     oc new-project linux-system-roles-test
     oc create -f openshift-objects.yml
 
-The service account needs to be in the privileged scc. Edit it with
+The service account needs to be in the privileged scc. If you have appropriate
+permissions, you can edit it with
 
     oc edit scc privileged
 
 and add `- system:serviceaccount:linux-system-roles:tester` under `users`.
 
+If you need to run container as root (for example for accessing `/dev/kvm`),
+apply following file:
+
+    oc replace -f openshift-objects-root.yml
```

Comment:
Alternatively, we can use `oc patch` command:
```sh
oc patch dc linux-system-roles --type json -p '[{ "op": "add", "path": "/spec/template/spec/securityContext/runAsUser", "value": "0" },{ "op": "remove", "path": "/spec/template/spec/containers/0/securityContext/fsGroup" },{ "op": "add", "path": "/spec/template/spec/volumes/0/secret/defaultMode", "value": "0400" },{ "op": "add", "path": "/spec/template/spec/volumes/1/configMap/defaultMode", "value": "0644" }]'
```
More info [here](https://labs.consol.de/development/2019/04/08/oc-patch-unleashed.html) and [here](http://erosb.github.io/post/json-patch-vs-merge-patch/).